plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
@DevinVinson You are free to decide. I am thinking about forking your repo and creating PHP 5.3 ready boilerplate with namespaces, autoloading etc. What would be the best way to do it and still give you credits for your work?